wireapp/wire-android-reloaded
# feat: Update Calling UI

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

[AR-1252 - Update UI](https://wearezeta.atlassian.net/browse/AR-1252)
[AR-1253 - Decline Incoming Call](https://wearezeta.atlassian.net/browse/AR-1253)

### Issues

When the user received a call, nothing happened 🤷🏻‍♂️ 

### Solutions

Now Incoming call UI is shown and you can accept and reject a call, if accepted then it will properly stablish a call with whoever is calling :D

### Dependencies (Optional)

Needs releases with:

- [X] #509 
- [X] #524 
- [X] #525 

#### How to Test

- Open App
- Send a message to the other user and receive a message from the other user (to make sure everything is working correctly in the app)
- Go back to Home screen
- from the other user call your client
- Incoming call UI should pop up

### Notes (Optional)

Some other things like audio permission will be handled later on in another PR

### Attachments (Optional)

<img src="https://user-images.githubusercontent.com/5890660/164199707-7bee79ec-b226-40a3-85fb-0cf67677e42f.jpeg" width="200" height="400" alt="incoming call ui" />

Video:

https://user-images.githubusercontent.com/5890660/164457123-917518ad-9f50-4733-b086-a655343e2ea2.mov
